src/loongarch64/sysv.S

```diff
@@ -147,7 +147,7 @@ ffi_call_asm:
 	.size	ffi_call_asm, .-ffi_call_asm
 
 
-/* ffi_closure_asm. Expects address of the passed-in ffi_closure in t1.
```

Comment:
Note: this is a document typo, which cost me half an hour to debug some "bus error".